src/Helpers/Form/Template.php

```diff
@@ -21,7 +21,13 @@ class Template
      */
     public static function getActiveID($formId = null)
     {
-        return Give()->form_meta->get_meta($formId ?: Frontend::getFormId(), '_give_form_template', true);
+        /*
+         * Bailout: If neither the parameter nor the Utils\Frontend allows to get a form ID
+         * do not call get_meta.
+         * This method return unpredictable values if called with an invalid ID.
+         */
+        if (!($formId = $formId ?: Frontend::getFormId())) return null;
```

Comment:
While `null` may be more correct in this situation, I'd rather not introduce a breaking change at this time. Changing return types can cause new and unforeseen problems.

If this is causing an issue in regards to some custom code then I would suggest introducing some defensive programming when calling this method.